parfive/config.py

```diff
@@ -0,0 +1,215 @@
+import os
+import sys
+import warnings
+from typing import Dict, Union, Optional
+
+try:
+    from typing import Literal  # Added in Python 3.8
+except ImportError:
+    from typing_extensions import Literal  # type: ignore
+
+from dataclasses import InitVar, field, asdict, dataclass
+
+import aiohttp
+
+import parfive
+from parfive.utils import ParfiveUserWarning
+
+__all__ = ["DownloaderConfig", "SessionConfig"]
+
+
+def _default_headers():
+    return {
+        "User-Agent": f"parfive/{parfive.__version__} aiohttp/{aiohttp.__version__} python/{sys.version[:5]}"
```

Comment:
On Python `3.9.13` `print(sys.version[:5])` gives me `3.9.1`, so it's probably better to use `platform.python_version()` that gives just the python version.